source/common/http/filter_manager.cc

```diff
@@ -480,6 +480,9 @@ void ActiveStreamDecoderFilter::sendLocalReply(
     Code code, absl::string_view body,
     std::function<void(ResponseHeaderMap& headers)> modify_headers,
     const absl::optional<Grpc::Status::GrpcStatus> grpc_status, absl::string_view details) {
+
+  setLocalReplyOwnerInformation();
+
```

Comment:
Sorry if I wasn't clear - but these two lines could be added as a method in `ActiveStreamFilterBase` that is then called from two subclasses. That would reduce duplication.

Reply:
understand.

Reply:
it is not possible to move the complete method to the base class because the virtual method comes from other base to the concrete class. I was not sure to use the same name, I didn't find a better one. please let me know if I should rename it